tests/batch/test_configuration_checker.py

```diff
@@ -9,7 +9,7 @@
 
 from dfastmi.io.RiversObject import RiversObject
 
-class Test_a_configuration_checker():
+class Test_configuration_checker():
```

Comment:
git compare online is a little bit hard to discern what has exactly changed here, maybe discuss it?

Reply:
is cleared up